projects/gis/src/app/app-routing.module.ts

```diff
@@ -1,23 +1,25 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
-import {IsLoggedInGuard} from '@geoengine/core';
+import {BackendStatusPageComponent, LogInGuard, NotFoundComponent, BackendAvailableGuard} from '@geoengine/core';
 import {CanRegisterGuard} from './guards/can-register.guard';
 import {LoginComponent} from './login/login.component';
 import {MainComponent} from './main/main.component';
 import {RegisterComponent} from './register/register.component';
 
 const routes: Routes = [
     {path: '', redirectTo: 'map', pathMatch: 'full'},
-    {path: 'map', component: MainComponent, canActivate: [IsLoggedInGuard]},
+    {path: 'map', component: MainComponent, canActivate: [BackendAvailableGuard, LogInGuard]},
     {path: 'signin', component: LoginComponent},
     {path: 'register', component: RegisterComponent, canActivate: [CanRegisterGuard]},
-    // seems to be a good fallback if we cannot acces `map`
-    {path: '**', redirectTo: 'signin', pathMatch: 'full'},
+    {path: '404', component: NotFoundComponent},
```

Comment:
Either remove or somehow make `**` re-route to 404.

For instance, http://localhost:4200/#/foo

Reply:
[c49a53e](https://github.com/geo-engine/geoengine-ui/pull/512/commits/c49a53e230d1eb8973ffd2c1157fe5df1c7c87f9)